java/vector/src/test/java/org/apache/arrow/vector/complex/writer/TestComplexWriter.java

```diff
@@ -266,6 +266,41 @@ public void listScalarType() {
     }
   }
 
+  @Test
+  public void testListScalarNull() {
```

Comment:
can you please add a small description for the test ? eg.

"Write to a integer list vector, each list of size 8 and having it's data values alternating between null and a non-null. Read and verify".

Reply:
done